src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/PartyFlagsMappingTest.java

```diff
@@ -360,17 +354,20 @@ void urgentCase(String urgentCase) {
     }
 
 
-    @DisplayName("adjournCaseInterpreterLanguage Parameterised Tests")
-    @ParameterizedTest
-    @ValueSource(strings = {"spanish", "french"})
-    void adjournCaseInterpreterLanguage(String interpreterLanguage) {
+    @DisplayName("adjournCaseInterpreterLanguage should return party flag mapping if interpreter required")
+    @Test
+    void adjournCaseInterpreterLanguage() {
         SscsCaseData caseData = SscsCaseData.builder()
             .adjournment(Adjournment.builder()
-                .interpreterLanguage(interpreterLanguage)
+                    .interpreterRequired(YES)
```

Comment:
This should also have adjournmentInProgress(YES) in order to be in effect.

Likewise, a test is required to show that when it is NO, this does not trigger.

Reply:
Test added for NO

Reply:
test added for No